src/definitions/elements/label.less

```diff
@@ -385,6 +385,26 @@ a.ui.label {
   padding-left: @ribbonTriangleSize;
   padding-right: @ribbonDistance;
 }
+.ui.basic.ribbon.label:not([class*="right ribbon"]) {
+  padding-top: calc(@verticalPadding - 1px);
+  padding-bottom: calc(@verticalPadding - 1px);
```

Comment:
For saver use of LESS not calculating values in brackets at compilation time (put make sure put the calc straight into CSS), please also move every `calc` values into
expressions, so for example
```less
padding-bottom: calc(@verticalPadding - 1px);
```
becomes
```less
padding-bottom:  e(%("calc(%d - %d)", @verticalPadding, @basicBorderWidth));
```

Reply:
And , to stay consistent with current codebase, even move that expression into a variable aswell as all the other ribbon settings already do:

```less
@basicRibbonOffset: e(%("calc(%d - %d)", @verticalPadding, @basicBorderWidth));
...
padding-bottom:  @basicRibbonOffset;
```

Reply:
Hopefully, better. It's my first time to use LESS. Thanks for your help so far.